python/lsst/ctrl/mpexec/cli/script/run.py

```diff
@@ -179,6 +180,8 @@ def run(  # type: ignore
     rebase : `bool`
         If `True` then reset output collection chain if it is inconsistent with
         the ``inputs``.
+    raise_on_partial_outputs : `bool`
+        Consider partial outputs and error instead of a success.
```

Comment:
Is this what you meant?
```suggestion
        Consider partial outputs an error instead of a success.
```